src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/controller/AgreementController.java

```diff
@@ -44,13 +65,40 @@
   }
 
   @GetMapping("/agreements/{ca-number}")
-  public AgreementDetail getAgreement(@PathVariable(value = "ca-number") String caNumber) {
+  public AgreementDetail getAgreement(@PathVariable(value = "ca-number") String caNumber) throws ParseException {
```

Comment:
Why is the ParseException not handled?

Reply:
Removed since we don't need it anymore (it was from the old code)